package.json

```diff
@@ -146,10 +147,10 @@
     "stylelint-config-prettier": "^6.0.0",
     "stylelint-prettier": "^1.1.1",
     "ts-node": "5.0.1",
-    "tsickle": "^0.37.0",
-    "tslint": "^6.1.3",
+    "tsickle": "^0.37.1",
+    "tslint": "~6.1.0",
     "tslint-plugin-prettier": "2.0.1",
-    "typescript": "^3.6.5",
+    "typescript": "^4.1.5",
```

Comment:
![giphub](https://media3.giphy.com/media/ps449csUx1X8s/giphy.gif)

Reply:
Yeah! Unfortunately I couldn't bump it all the way, but this is great for now